platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MinimumJudgeInfo.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2016-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.state;
+
+import com.swirlds.common.io.streams.SerializableDataInputStream;
+import com.swirlds.common.io.streams.SerializableDataOutputStream;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Records the minimum ancient threshold for all judges in a particular round.
```

Comment:
Wouldn't it be the minimum ancient `indicator`?   The minimum indicator of  judges in round `latestConsensusRound-26+1` becomes the ancient threshold.    This is storing the minimum indicator for the given round, not the minimum threshold for round -26+1

Reply:
changed to `indicator`